microsoft-edge/devtools-guide-chromium/whats-new/2023/08/devtools-116.md

```diff
@@ -0,0 +1,115 @@
+
+
+<!-- ====================================================================== -->
+## "Move Activity Bar" button in the Activity bar
+
+<!-- Subtitle: __ -->
+<!-- todo: need Subtitle sentence -->
+
+While in Focus Mode, you now have the ability to easily move the Activity bar's location with a click of a button.  In the upper left of DevTools, to the right of the **Inspect** button and **Device Emulation** button, there's now a **Move Activity Bar to left** or **Move Activity Bar to top** button.  The new button is always available in the Activity bar, instead of having to go into the DevTools **Settings** (...) menu.
+
+For example, by default, the Activity bar's orientation is horizontal, but you can change it to vertical by simply clicking the new **Move Activity Bar to left** button:
+
+![Moving the location of the Activity Bar in Focus Mode](./devtools-116-images/move-location-activity-bar.gif)
+
+See also:
+* [What's New entries about Focus Mode](/search/?scope=Microsoft%20Edge&terms=focus%20mode)
+<!--
+* [Dec. 2023 blog post about Focus Mode]() - todo: link -->
+
+
+<!-- ====================================================================== -->
+## React Developer Tools extension retains state of tools in Focus Mode
+
+<!-- Subtitle: The React Developer Tools extension retains the state of tools while in Focus Mode. -->
+<!-- todo: write actual subtitle -->
+
+Prior to Microsoft Edge 116, if you were in Focus Mode while using the React Developer Tools extension, the state of a tool in DevTools wasn't maintained when you switched to another tool and then switched back.  This issue has been resolved.
+
+See also:
+* [React Developer Tools](https://microsoftedge.microsoft.com/addons/detail/react-developer-tools/gpphkfbcpidddadnkolkpfckpihlkkil) - extension at the Edge Add-ons store
+* [What's New entries about Focus Mode](https://learn.microsoft.com/search/?scope=Microsoft%20Edge&terms=focus%20mode)
+<!--
+* [Dec. 2023 blog post about Focus Mode]() - todo: link -->
+
+
+<!-- ====================================================================== -->
+## Containment Diff view in Memory tool when multiple heap snapshots
+
+<!-- Subtitle: __ -->
+<!-- todo: need Subtitle sentence -->
+
+When there's more than one heap snapshot, the "Containment Diff" view will be available in the View dropdown:
+
+![The Containment Diff view in the Memory tool](./devtools-116-images/new-view-memory-panel.gif)
+
+The Containment Diff view shows only the new objects, deleted objects, or objects in the retaining chain that will eventually lead to either a new or deleted node.  The "." symbol means that it's a node in the retaining chain.  Expanding the view displays the node that's new or deleted.
+
+See also:
+* [Record heap snapshots using the Memory tool](../../../memory-problems/heap-snapshots.md)
+
+
+<!-- ====================================================================== -->
+## Added an icon in the Recorder tool's tab
+
+<!-- Subtitle: __ -->
```

Comment:
add Subtitle sentence for entry: Added an icon in the Recorder tool's tab
@leahmsft

Reply:
won't do since we won't add to welcome tool